zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneController.java

```diff
@@ -110,4 +112,29 @@ public ResponseEntity<OrchestratorErrorResponse> internalServerError(Exception e
     {
         return ResponseEntity.status(INTERNAL_SERVER_ERROR).body(new OrchestratorErrorResponse(ex.getMessage()));
     }
+
+    @ExceptionHandler(value = { HttpMessageNotReadableException.class })
+    public ResponseEntity<OrchestratorErrorResponse> messageReadableException(HttpMessageNotReadableException ex)
+    {
+        return ResponseEntity.status(BAD_REQUEST).body(new OrchestratorErrorResponse(ErrorMessageUtil.getErrorDetail(ex)));
+    }
+
+    @ExceptionHandler(value = { MethodArgumentNotValidException.class })
+    public ResponseEntity<OrchestratorErrorResponse> methodArgumentException(MethodArgumentNotValidException ex) {
+        BindingResult bindingResult = ex.getBindingResult();
+        Set<String> errorMessages = new HashSet<>();
+        for (ObjectError objectError: bindingResult.getAllErrors()) {
+            String message = ErrorMessageUtil.getAffectedProperty(objectError) + " "+ objectError.getDefaultMessage();
+            errorMessages.add(message);
+        }
+        return ResponseEntity.status(BAD_REQUEST).body(new OrchestratorErrorResponse(String.join("; ",
+                                                                                                 errorMessages)));
+    }
+
+    @ExceptionHandler(value = { ConstraintViolationException.class})
+    public ResponseEntity<OrchestratorErrorResponse> constraintViolationException(ConstraintViolationException ex)
+    {
+        String errorMessage = ex.getMessage().replaceAll(GET_ZONE_PREFIX, "");
```

Comment:
May be handle  ConstraintViolationException in this way https://stackoverflow.com/questions/36555057/get-field-name-when-javax-validation-constraintviolationexception-is-thrown and https://www.baeldung.com/global-error-handler-in-a-spring-rest-api.

Reply:
Done, Thanks!
which will retrun error response as {
    "message": "name must be specified"
} for both get and delete zone.
Also added a integration test for testDeleteZone_NameRequiredError()